me.sanchezrodriguez.passes.json

```diff
@@ -0,0 +1,45 @@
+{
+    "app-id" : "me.sanchezrodriguez.passes",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "passes",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
```

Comment:
does it really access to the whole home directory?

Reply:
I removed the permission from the application repository but I forgot to do it from this one. Updated.